### ISSUE TITLE
Account with ENS name is shown both with and without identicon ring

### ISSUE THREAD

**Issue by glitchminer** [status-im/status-desktop#14049]
# Bug Report

## Description

Add a contact that uses ENS
Note that messages are shown without identicon ring but account still displays it

![image](https://github.com/status-im/status-desktop/assets/50769329/307b7ead-1655-45ee-a282-6ea4bb5e4232)

### Additional Information

- Status desktop version: Master
- Operating System: Mac

**Comment by caybro** [comment]
@glitchminer can you pls post bigger/better screenshot(s)? hard to tell what is what :)

**Comment by anastasiyaig** [comment]
@alexandraB99 please check on milestone, where should we fix that? Is it for rc?

**Comment by glitchminer** [comment]
> @glitchminer can you pls post bigger/better screenshot(s)? hard to tell what is what :)

Yeah, sorry about that. The top one with the identicon ring showing when it should not is the header of the chat. The lower one is an individual message in the chat. 

This is from the perspective of a contact having a chat with a user that is using an ENS name. The bug is that the identicon ring appeared in chat header and contacts list when it should not have.

**Comment by caybro** [comment]
> > @glitchminer can you pls post bigger/better screenshot(s)? hard to tell what is what :)
> 
> Yeah, sorry about that. The top one with the identicon ring showing when it should not is the header of the chat. The lower one is an individual message in the chat.
> 
> This is from the perspective of a contact having a chat with a user that is using an ENS name. The bug is that the identicon ring appeared in chat header and contacts list when it should not have.

got it... but can't reproduce :/
